attribution-graphs-frontend/attribution_graph/cg.css

```
@@ -0,0 +1,772 @@
+            line-height: 1em;
+
+            th, td {
+                text-align: center;
+                font-size: 11px !important;
+                border-bottom-color: #fff !important;
+                line-height: 1em;
+            }
+
+            th:not(:first-child), td:not(:first-child){
+                padding-left: 0px !important;
+                padding: 0px 10px !important;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-left



(lint/suspicious/noShorthandPropertyOverrides)